src/entities/transfer-process.ts

```diff
@@ -17,6 +17,17 @@ export class TransferProcess extends JsonLdId {
   dataDestination?: { properties: { type: string } };
 }
 
+export class TransferProcessState extends JsonLdId {
+  state?: TransferProcessStates;
+}
+
+export enum TransferProcessStates {
+  INITIAL = "INITIAL",
+  FINALIZED = "FINALIZED",
+  STARTED = "STARTED",
+  REQUESTING = "REQUESTING",
+}
+
```

Comment:
@ndr-brt I wasn't able to locate any state machine diagram or similar resource on the official repo and those were just based on the connector logs. Please let me know if I forgot any state and I will include it before getting this merged.

Reply:
@OlfaBensoussia here you can find it: https://eclipse-edc.github.io/docs/#/submodule/Connector/docs/developer/architecture/usage-control/contracts?id=state-machine

yes, there are some states missing, you can get a complete list here: https://github.com/eclipse-edc/Connector/blob/6c2aa059409cc03c72ac86bfee5ea7429c6367aa/spi/control-plane/transfer-spi/src/main/java/org/eclipse/edc/connector/transfer/spi/types/TransferProcessStates.java#L44